tools/patchformat.py

```diff
@@ -0,0 +1,48 @@
+import argparse
+from ghapi.core import GhApi
+import os
+import time
+
+username = os.environ['GH_USERNAME']
+github_token = os.environ['GH_TOKEN']
+
+api = GhApi(owner='kubernetes-security', repo='kubernetes',token=github_token)
+
+parser = argparse.ArgumentParser(description='Format patch files')
+required = parser.add_argument_group('required arguments')
+required.add_argument('--pr', help='Pull Request Number in kubernetes-security/kubernetes with the fix (e.g. 63)', required=True)
+required.add_argument('--cve', help='CVE for the vulnerability', required=True)
+required.add_argument('-r', '--release', help='Kubernetes release branch (e.g. 1.30)', required=True)
+parser.add_argument('-v', '--version', help='Patch version, if a correction is being sent')
+args = parser.parse_args()
+
+pull = api.pulls.get(args.pr)
```

Comment:
I'm getting 404 with my token. I wonder if I'm missing certain permissions off the repo? Lets pair on this when you have time.

Reply:
I checked and you have admin on the repo but maybe the token doesn't have permission? I'm back from vacation this week and can pair when you're free!